bcda/web/middleware_test.go

```diff
@@ -151,7 +151,9 @@ func (s *MiddlewareTestSuite) TestHSTSHeader() {
 	router.ServeHTTP(w, req)
 	result := w.Result()
 
-	assert.NotEmpty(s.T(), result.Header.Get("Strict-Transport-Security"), "sets HSTS header")
+	assert.NotEmpty(s.T(), result.Header.Get("Strict-Transport-Security"), "sets Security header")
+	assert.Contains(s.T(), result.Header.Get("Cache-Control"), "must-revalidate", "ensures must-revalidate control added")
```

Comment:
I agree that this addresses the ticket.  But, from a redundancy perspective, should we ensure that `Cache-Control` also contains: `max-age=0, no-cache, no-store`?
Right now, we are getting that from Akamai, but we should also bake it into the app.
@dhgreene Thoughts?

Reply:
@msnook @jeffcorpuz That does seem prudent.